spec/models/agents/twitter_user_agent_spec.rb

```diff
@@ -42,4 +43,21 @@
     end
   end
 
+  describe "#check that home timeline works" do
+    before do
+      stub_request(:any, //).to_return(:body => File.read(Rails.root.join("spec/data_fixtures/user_tweets.json")), :status => 200)
```

Comment:
It would be nice to specify which request is stubbed. We have the problem that `VCR` and `webmock` are not playing nice together, when you comment out or remove [the VCR configuration](https://github.com/cantino/huginn/blob/master/spec/support/vcr_support.rb) temporarily to run the twitter user agent specs webmock should print more information about the request, which you can then copy and paste.